### PR TITLE
torch-L2RPN

### PR DESCRIPTION
**Pull request by YuechengLiu**

### CODE REVIEW

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Dependencies
+- python3.6
+- [parl==1.3.2](https://github.com/PaddlePaddle/PARL)
+- Pytorch
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/README.md]: Please specify the version here.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        msg_tmp += "\t\t - number of time steps completed: {:.0f} / {:.0f}".format(
+            nb_time_step, max_ts)
+        print(msg_tmp)
+
+    if save_gif:
+        save_log_gif(logs_path, res)
+
+
+if __name__ == "__main__":
+    """
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py:108]: Remove this comment. It does not provide any information.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+import torch
+import os
+from grid2op.Runner import Runner
+from l2rpn_baselines.utils.save_log_gif import save_log_gif
+from rl_agent import RLAgent
+import argparse
+
+
+def cli():
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py]: cli -> parse_args

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+import torch
+import os
+from grid2op.Runner import Runner
+from l2rpn_baselines.utils.save_log_gif import save_log_gif
+from rl_agent import RLAgent
+import argparse
+
+
+def cli():
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py]: rename this function.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    parser.add_argument(
+        "--nb_episode",
+        required=False,
+        default=1,
+        type=int,
+        help="Number of episodes to evaluate")
+    parser.add_argument(
+        "--nb_process",
+        required=False,
+        default=1,
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py:46]: Have you tested running the script with nb_process > 1?

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+        msg_tmp += "\t\t - number of time steps completed: {:.0f} / {:.0f}".format(
+            nb_time_step, max_ts)
+        print(msg_tmp)
+
+    if save_gif:
+        save_log_gif(logs_path, res)
+
+
+if __name__ == "__main__":
+    """
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py:108]: Remove this comment.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+    This is a possible implementation of the eval script.
+    """
+    import grid2op
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py]: Why don't you import them at the header ?

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/rl_agent.py [hunk 8]
```diff
@@ -0,0 +1,517 @@
...
+        for i in range(redispatch_actions_vec.shape[0]):
+            action = action_space.from_vect(redispatch_actions_vec[i])
+            self.redispatch_actions.append(action)
+
+        with open(
+                os.path.join('saved_files', "action_to_sub_id.pickle"),
+                "rb") as f:
+            self.action_to_sub_id = pickle.load(f)
+
+        self.after_line56_disconnect_actions = []
```
Comment (zenghsh3) [inline benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/rl_agent.py]: rename `after_line56_disconnect_actions` to `after_line56_or_line45_disconnect_actions`.
rename it in the paddle version at the same time.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py (hunk 7 above)
Comment (YuechengLiu) [reply benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py]: copied from origin template. modified.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/README.md (hunk 1 above)
Comment (YuechengLiu) [reply benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/README.md]: updated

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py (hunk 2 above)
Comment (YuechengLiu) [reply benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py:108]: removed.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py (hunk 3 above)
Comment (YuechengLiu) [reply benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/evaluate.py]: updated

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py (hunk 5 above)
Comment (YuechengLiu) [reply benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py:46]: tested.

File: benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/rl_agent.py (hunk 8 above)
Comment (YuechengLiu) [reply benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track1/rl_agent.py]: updated

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**